docs/source/admin/quick_howto/ciab.rst

```diff
@@ -310,3 +310,16 @@ Dante's socks proxy is an optional container that can be used to provide browser
 #. Once the CDN-in-a-box stack has started, use the aforementioned browser to access traffic portal via the socks proxy on the docker host.
 
 .. seealso:: `The official Docker Compose documentation CLI reference <https://docs.docker.com/compose/reference/>`_ for complete instructions on how to pass service definition files to the ``docker-compose`` executable.
+
+Static Subnet
+-------------
+Since ``docker-compose`` will randomly create a subnet and it has a chance to conflict with your network environment, using static subnet is a good choice.
+
+.. code-block:: shell
+	:caption: CIAB Startup Using Bash Alias
```

Comment:
Captions should be unique. This does more than just "start up CDN-in-a-Box", it's doing it with a static subnet.

Reply:
Sorry that there are tons of duplicate caption so I didn't aware that. I have fixed it.
Thanks!